sdk/src/driftClient.ts

```diff
@@ -255,15 +255,21 @@ export class DriftClient {
 	}
 
 	public async subscribe(): Promise<boolean> {
-		let subscribePromises = this.subscribeUsers().concat(
-			this.accountSubscriber.subscribe()
-		);
+		let subscribePromises = [
+			this.addAndSubscribeToUsers(
```

Comment:
all the parameters for this function exist on the DriftClient, do you need to pass em as parameters?